server/src/com/cloud/network/router/VirtualNetworkApplianceManagerImpl.java

```diff
@@ -1971,7 +1971,7 @@ protected void finalizeNetworkRulesForNetwork(final Commands cmds, final DomainR
     }
 
     private void createDefaultEgressFirewallRule(final List<FirewallRule> rules, final long networkId) {
-        String systemRule = null;
+        // String systemRule = null;
```

Comment:
While making changes in this file, I noticed that Eclipse was pointing out that the variable systemRule was only written to (never read from). Instead of removing the entire line, I just commented it out in case someone later wanted to easily resume making use of this variable.